include/ddc/uniform_discretization.hpp

```diff
@@ -124,6 +124,18 @@ Coordinate<CDim> to_real(DiscreteCoordinate<UniformDiscretization<CDim>> const&
     return discretization<UniformDiscretization<CDim>>().to_real(c);
 }
 
+template <class CDim>
+Coordinate<CDim> distance_at_left(DiscreteCoordinate<UniformDiscretization<CDim>> i)
+{
+    return discretization<UniformDiscretization<CDim>>().step();
+}
+
+template <class CDim>
+Coordinate<CDim> distance_at_right(DiscreteCoordinate<UniformDiscretization<CDim>> i)
```

Comment:
maybe add a `max_step_between()`

Reply:
I am not against, but I would wait for the need for it. We could also provide a `distance_between()` algorithm.

Reply:
I added these two functions because the PR is about updating the example, not to provide new features on DDC. I can move the functions to the example if needed.

Reply:
works for me like that :+1: no need to change